lib/telegraph/client.rb

```diff
@@ -0,0 +1,67 @@
+module Telegraph
+  class Client
+    include Telegraph::Connection
+
+    attr_reader :token
+
+    def initialize(token)
+      @token = token
+    end
+
+    def create_account(attrs = {})
+      acc_info = get('createAccount', attrs)
```

Comment:
Despite the fact that API allows both GET and POST, I'd say methods that change something should go as POST always.